mabl snippets/variableEqualityComparison.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Using mablInputs to define the path of a URL.
+ * @param {object} - mablInputs Object containing input
+ *                   variables (mablInputs.variables.user)
+ * @param {function} callback - The callback function
+ */
+function mablJavaScriptStep(mablInputs, callback) {
+  
+    // grab any variable here that you would like to compare against a value or another variable
+    let var1 = mablInputs.variables.user.var1;
+
+    // variable 2 to use in comparison
+    let var2 = mablInputs.variables.user.var2; 
+
+    if (var1 === null || var2 === null) {
```

Comment:
more likely to be `undefined` than `null`, so should probably check for that (instead or in addition)